packages/grafana-ui/src/components/Stack/Stack.internal.story.tsx

```diff
@@ -31,4 +33,176 @@ export const Basic: StoryFn<typeof Stack> = ({ direction = 'column', gap = 2 })
   );
 };
 
+export const TestCases: StoryFn<typeof Stack> = ({ direction = 'column', gap = 2 }) => {
```

Comment:
I'm not sure what you are trying to bring across in this story yet. Is it showing examples of how not to use Stack (the word mismatched occurs quite a lot)? 
It could make it more obvious by putting wrong and right next to each other (if this is what you want to show) or consider another kind of grouping.

Reply:
Good point